homeassistant/components/vera/__init__.py

```diff
@@ -70,7 +70,8 @@ def setup(hass, base_config):
     def stop_subscription(event):
         """Shutdown Vera subscriptions and subscription thread on exit."""
         _LOGGER.info("Shutting down subscriptions")
-        hass.data[VERA_CONTROLLER].stop()
+
+        hass.add_job(hass.data[VERA_CONTROLLER].stop)
```

Comment:
Why is this needed? The `stop_subscription` callback has already been added as a job.